src/test/java/kitchenpos/menu/acceptance/MenuAcceptance.java

```diff
@@ -0,0 +1,48 @@
+package kitchenpos.menu.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuProduct;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MenuAcceptance {
```

Comment:
```suggestion
public class MenuTestFixture {
```

테스트용 리소스를 생성하는 역할은 `TextFixture` 라는 명칭을 주로 사용합니다.